jetty-server/src/test/java/org/eclipse/jetty/server/ForwardedRequestCustomizerTest.java

```diff
@@ -82,6 +84,15 @@ public void init() throws Exception
         connector = new LocalConnector(server, http);
         server.addConnector(connector);
 
+        // Alternate behavior Connector
+        HttpConnectionFactory httpAlt = new HttpConnectionFactory();
+        httpAlt.setInputBufferSize(1024);
```

Comment:
I wonder why we are setting the input buffer  size in this test?

Reply:
no clue, i'll remove it and see what happens